scripts/api_master.py

```diff
@@ -874,6 +961,7 @@
                         "keras.ops.bincount",
                         "keras.ops.broadcast_to",
                         "keras.ops.ceil",
+                        "keras.ops.cholesky"
```

Comment:
We need to create a new category for linalg ops, with all ops from `keras.ops.linalg`.

Reply:
Fixed! Moved all keras.ops.linalg ops to new section and removed them from core/numpy